provider/resources.go

```diff
@@ -128,67 +130,145 @@ func Provider() tfbridge.ProviderInfo {
-			"dynatrace_xmatters_notification":      {Tok: dynatraceResource(mainMod, "XmattersNotification")},
+			"dynatrace_alerting":                      {Tok: dynatraceResource(mainMod, "Alerting")},
+			"dynatrace_activegate_token":              {Tok: dynatraceResource(mainMod, "ActivegateToken")},
+			"dynatrace_activegate_updates":            {Tok: dynatraceResource(mainMod, "ActivegateUpdates")},
+			"dynatrace_alerting_profile":              {Tok: dynatraceResource(mainMod, "AlertingProfile")},
+			"dynatrace_aix_extension":                 {Tok: dynatraceResource(mainMod, "AixExtension")},
+			"dynatrace_application_anomalies":         {Tok: dynatraceResource(mainMod, "ApplicationAnomalies")},
+			"dynatrace_application_data_privacy":      {Tok: dynatraceResource(mainMod, "ApplicationDataPrivacy")},
+			"dynatrace_application_detection_rule":    {Tok: dynatraceResource(mainMod, "ApplicationDetectionRule")},
+			"dynatrace_application_error_rules":       {Tok: dynatraceResource(mainMod, "ApplicationErrorRules")},
+			"dynatrace_api_detection":                 {Tok: dynatraceResource(mainMod, "ApiDetection")},
+			"dynatrace_api_token":                     {Tok: dynatraceResource(mainMod, "ApiToken")},
+			"dynatrace_application_detection_rule_v2": {Tok: dynatraceResource(mainMod, "ApplicationDetectionRuleV2")},
+			"dynatrace_ansible_tower_notification":    {Tok: dynatraceResource(mainMod, "AnsibleTowerNotification")},
+			"dynatrace_autotag":                       {Tok: dynatraceResource(mainMod, "Autotag")},
+			"dynatrace_aws_credentials":               {Tok: dynatraceResource(mainMod, "AwsCredentials")},
+			"dynatrace_audit_log":                     {Tok: dynatraceResource(mainMod, "AuditLog")},
+			"dynatrace_autotag_v2":                    {Tok: dynatraceResource(mainMod, "AutotagV2")},
+			"dynatrace_aws_anomalies":                 {Tok: dynatraceResource(mainMod, "AwsAnomalies")},
+			"dynatrace_azure_credentials":             {Tok: dynatraceResource(mainMod, "AzureCredentials")},
+			"dynatrace_browser_monitor":               {Tok: dynatraceResource(mainMod, "BrowserMonitor")},
+			"dynatrace_browser_monitor_outage":        {Tok: dynatraceResource(mainMod, "BrowserMonitorOutage")},
+			"dynatrace_browser_monitor_performance":   {Tok: dynatraceResource(mainMod, "BrowserMonitorPerformance")},
+			"dynatrace_business_events_buckets":       {Tok: dynatraceResource(mainMod, "BusinessEventsBuckets")},
+			"dynatrace_business_events_metrics":       {Tok: dynatraceResource(mainMod, "BusinessEventsMetrics")},
+			"dynatrace_business_events_oneagent":      {Tok: dynatraceResource(mainMod, "BusinessEventsOneagent")},
+			"dynatrace_business_events_processing":    {Tok: dynatraceResource(mainMod, "BusinessEventsProcessing")},
+			"dynatrace_calculated_service_metric":     {Tok: dynatraceResource(mainMod, "CalculatedServiceMetric")},
+			"dynatrace_cloudfoundry_credentials":      {Tok: dynatraceResource(mainMod, "CloudfoundaryCredentials")},
```

Comment:
```suggestion
			"dynatrace_cloudfoundry_credentials":      {Tok: dynatraceResource(mainMod, "CloudfoundryCredentials")},
```

Typo in the original mapping.